lib/commands/download-translations.js

```diff
@@ -22,36 +23,47 @@ async function downloadTranslations(shouldIgnoreUnmergedMaster = false) {
   await uploadSources();
 
   console.log('Crowdin: Triggering branch build before downloading');
-  await CrowdinApi.buildBranch(crowdinBranchName);
+  const branchData = await CrowdinApi.getBranchData(crowdinBranchName);
+  const buildData = await CrowdinApi.buildBranch(branchData.data.id);
 
+  let buildStatusData;
+
+  do {
+    buildStatusData = await CrowdinApi.checkBuildStatus(buildData.data.id);
+
+    console.log('Crowdin: Build progress:', buildStatusData.data.progress);
+  } while (buildStatusData && buildStatusData.data.status !== 'finished');
```

Comment:
let's add constants for statuses and other hardcoded strings?

Reply:
@bolelamx Sure, thanks for reminding!